tests/gis_tests/geoapp/tests.py

```diff
@@ -465,6 +467,18 @@ def test_relate_lookup(self):
             self.assertEqual('Texas', Country.objects.get(mpoly__relate=(pnt2, intersects_mask)).name)
             self.assertEqual('Lawrence', City.objects.get(point__relate=(ks.poly, intersects_mask)).name)
 
+    def test_geometry_value_annotation(self):
```

Comment:
The location still doesn't seem ideal as the new tests don't use `__` lookups like the others. Maybe a new `test_expressions.py` file that includes all 3 added tests would make sense?